### PR TITLE
Fix Default column for core tokens

### PR DESCRIPTION
**Pull request by deivamagalhaes**
# Summary

This PR fixes the Default column so it displays the "Default" mark for FW tokens and core tokens alike. 

## Details

Because the pre-existing filter expects a FW token as a param, it is only applied if the token is a FW token.

## UI Changes

- The Default column displays the mark for core tokens as well: https://cloud.skyver.ge/lluyzZpD

### Setup

- Have at least one FW payment method saved
- Have at least one core payment method saved (recommended gateway: Stripe)

### Steps

1. Navigate to Payment Methods
1. Make the core token the default, if needed
    - [x] The "Default" column for the core token shows the Default mark
    - [x] The "Default" column for the FW token does not show the Default mark
1. Make the FW token the default
    - [x] The "Default" column for the core token does not show the Default mark
    - [x] The "Default" column for the FW token shows the Default mark

## Before merge

- [ ] I have confirmed these changes in each supported minor WooCommerce version
- [ ] I have copied all UI Changes and QA items listed above into the base release branch PR description

### CODE REVIEW

File: woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php [hunk 1]
```diff
@@ -857,22 +854,28 @@ protected function get_payment_method_title_html( SV_WC_Payment_Gateway_Payment_
 	 *
 	 * @since 5.1.0
 	 *
-	 * @param SV_WC_Payment_Gateway_Payment_Token $token token object
+	 * @param boolean $is_default true if the token is the default token
+	 * @param SV_WC_Payment_Gateway_Payment_Token|null $token FW token object, only set if the token is a FW token
 	 * @return string
 	 */
-	protected function get_payment_method_default_html( SV_WC_Payment_Gateway_Payment_Token $token ) {
+	protected function get_payment_method_default_html( $is_default, SV_WC_Payment_Gateway_Payment_Token $token = null ) {
```
Comment (asseti6) [inline woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php]: @deivamagalhaes assuming our min WP/WC PHP version does not support type casting here?  If it does this bool really should be type casted and set to a default of false.

Comment (deivamagalhaes) [reply woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php]: We support PHP 5.6, so yes, no scalar type declarations for us. I can still set the default though, if you think that would be helpful.

Comment (asseti6) [reply woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php]: Yeah I think we should default to that on a bool parameter.  Can easily have someone do a strict comparison later and be unaware.

Comment (asseti6) [reply woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php]: Ah my spelling sucks.. lets try this again: Yeah I think we should default to that on a bool parameter for readability. And it saves a step keeping things clear in the future where we do upgrade PHP versions.

**Pull request merged**